lnrpc/wtclientrpc/wtclient.go

```diff
@@ -288,7 +290,9 @@ func (c *WatchtowerClient) ListTowers(ctx context.Context,
 		rpcTowers[tower.ID] = rpcTower
 	}
 
-	legacyTowers, err := c.cfg.Client.RegisteredTowers(opts...)
+	legacyTowers, err := c.cfg.Client.RegisteredTowers(
+		req.ExcludeExhaustedSessions, opts...,
```

Comment:
shouldn't this be `!req.ExcludeExhaustedSessions`?

Reply:
indeed! thanks :)